web/js/app.js

```diff
@@ -24,8 +24,14 @@ $(document).ready(function() {
                     if (click2call_enable === true && value.type === 'tel' && item[key]) {
                         icon = '<i class="phone" data-tel="' + convertToTel(item[key]) + '"></i>';
                     }
+                    
+                    if (value.type === 'email' && item[key]) {
+                        tr += sprintf('<td class="%s"><a href="mailto:%s">%s</a></td>', value.type, item[key], item[key]);
```

Comment:
item[key] must be secured. With this code, you can have a XSS.
Use htmlspecialchars to secure it